english/flann-o-brien/truth-is-odd.md

```diff
@@ -0,0 +1,3 @@
+Becoming increasingly aware that the situation is more odd than even, you consult the legendary Irish writer Flann O'Brien, and his novel at-swim-two-birds, where you find the following: "there is safety in a triad, chastity is truth and truth is an odd 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.